source/docs/hardware/sensors/encoders-hardware.rst

```diff
@@ -36,14 +36,14 @@ On-shaft encoders
 
 .. todo:: add picture
 
-On-shaft encoders (such as the `AMT103-V <https://www.digikey.com/product-detail/en/cui-inc/AMT103-V/102-1308-ND/827016>`__ available through FIRST Choice) couple to a shaft by fitting *around* it, forming a friction coupling between the shaft and a rotating hub inside the encoder.
+On-shaft encoders (such as the `AMT103-V <https://www.cui.com/product/motion/rotary-encoders/incremental/modular/amt10-series>`__ available through FIRST Choice) couple to a shaft by fitting *around* it, forming a friction coupling between the shaft and a rotating hub inside the encoder.
```

Comment:
Why this change? That is not the AMT103-V

Reply:
Linkcheck was having problems with the [digikey link](https://www.digikey.com/product-detail/en/cui-inc/AMT103-V/102-1308-ND/827016), so I wanted to go straight to the manufacturer. I'll swap it for a link to the [AMT103-V](https://www.cui.com/product/motion/rotary-encoders/incremental/modular/amt10-v-kit/amt103-v)